unity/EditorPlugin/ILRepack.targets

```diff
@@ -1,11 +1,11 @@
 ﻿<Project>
   <ItemGroup>
+    <!-- Note that EditorPlugin.csproj adds Newtonsoft.Json.dll -->
     <ILRepackInputAssemblies Include="$(OutputPath)$(AssemblyName).dll" />
-    <ILRepackInputAssemblies Include="$(OutputPath)Newtonsoft.Json.dll" />
     <ILRepackInputAssemblies Include="../lib/*.dll" />
   </ItemGroup>
   <PropertyGroup>
-    <ILRepackOutputAssembly>../build/$(AssemblyName)/Assets/Plugins/Editor/JetBrains/$(AssemblyName).Repacked.dll</ILRepackOutputAssembly>
+    <ILRepackOutputAssembly>$(OutputPath)$(AssemblyName).Repacked.dll</ILRepackOutputAssembly>
```

Comment:
I guess this would produce Repacked dll in a different folder.
https://github.com/JetBrains/resharper-unity/blob/bace54c147b95458556368348d942a6715d5f301/unity/Directory.Build.props#L23